_artifacts/reports/ast-grep-inline-tests/summary_20250119.md

```
@@ -0,0 +1,117 @@
+# AST-grep Inline Test False Positives - Fix Summary
+
+## Issue
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)